tests/tests_cluster_basic_cloud_packages.yml

```diff
@@ -0,0 +1,69 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Minimal cluster configuration, install cloud agents
+  hosts: all
+  vars_files: vars/main.yml
+  vars:
+    ha_cluster_cluster_name: test-cluster
+    ha_cluster_install_cloud_agents: true
+    __test_agents_rhel_8:
+      - resource-agents-aliyun
+      - resource-agents-gcp
+      - fence-agents-aliyun
+      - fence-agents-aws
+      - fence-agents-azure-arm
+      - fence-agents-gce
+    __test_agents_rhel_9:
+      - resource-agents-cloud
+      - fence-agents-aliyun
+      - fence-agents-aws
+      - fence-agents-azure-arm
+      - fence-agents-compute
+      - fence-agents-gce
+      - fence-agents-ibm-powervs
+      - fence-agents-ibm-vpc
+      - fence-agents-kubevirt
+      - fence-agents-openstack
+    __test_agents: "{{ (ansible_facts['distribution_major_version'] == '8') |
+      ternary(__test_agents_rhel_8, __test_agents_rhel_9) }}"
+
+  tasks:
+    - name: Run test
+      tags: tests::verify
+      when: ansible_facts['distribution'] == "RedHat"
```

Comment:
What makes this RHEL specific?  Are the packages available in CentOS?

Reply:
My bad, somehow I thought this would match CentOS as well. I confused it with `os_family`. Fixed.

Reply:
Yeah, I wish there were an Ansible fact like `ansible_facts["os_family_and_version_compatible"] == "RedHat"` which would apply to RedHat, CentOS, Rocky, Alma, etc. but not Fedora and other distributions which are not roughly version compatible.